src/main/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
What's going on here? I'm guessing you're trying to merge that hyphen. Delete it, this should not be showing up in your commit.